### DISCUSSION TITLE
ender3v2 422 bltouch x-axis out

### DISCUSSION THREAD

**Discussion by lauraneb** [mriscoc/Ender3V2S1#1324] [General]
hello
I have an ender 3v2 422 card with bl touch
I installed firmware Ender3V2-422-BLTUBL-MPC-20240125.bin
the basic dimensions are good in the new firmware. and the bl touch flashes normally.
if I do a manual Home, autohome or a bed traming wizard the head goes to the right when wanting to come out of the frame and the motor forces.
Being a beginner, I don't understand what I'm doing wrong or what I should do to prevent this from happening again.

when the bed traming wizard command warns that a point is not good, it does not say what to do and does not allow it to be modified directly.
Thank you for your answers.

**Comment by mriscoc** [comment]
Check your mesh insets: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Reply by lauraneb** [reply]
I did the calibration guide:
from the advanced/phycical settings by checking my x,y,z dimensions with prepare/move axis. xmin=0 ymin0 zmin= ? xmax=247 ymax=233 zmax=250
I saved my changes with store settings.
I launched an autoHome which places the bltouch in the center of the bed (x=156.5 y=122 z=5) and stops. it does not return to x0 y0.
I placed my coordinates parkhead at x=0 y=0 z=20

after that I have to find my z offset as said on the bltouch 3d page which invites me before to do a bed tramming

but if I run the bed tramming / tramming wizard command it starts from the last autohome point (center of the board) and goes to the front center of the bed, takes a point, goes to the right as if it wanted to get out of the frame by forcing the motor , takes a point, and draws a triangle on half the board to take the back point.

I don't know what mistake I'm making so that he only makes his tramming bed on half the board.

to check I brought my head manually to x=0 and y=0 and I restarted an autohome which placed it back in the center of the bed.
But if I restart an autohome again it starts from the coordinates of the center of the bed and tries to move from this point and forces on the x axis wanting to exit the frame of the machine.
if I manually replace the head at x=0,y=0 and I launch bed tramming wizard then it goes to the center of the bed, takes a point, returns to the front left takes a point, goes to the front right takes a point, goes to the right rear and takes a point, without forcing the axes. 
thank to answers

**Reply by mriscoc** [reply]
You must complete the other steps indicated in the link above.

**Reply by yoshoiv** [reply]
i had the same issue, then i restarted my computer and downloaded the file again and then re-flashed it. try restarting your computer.
(note: this problem will also happen if you flash the printer with the sd card that comes with it when you buy it that card bugs the software even if you get it to flash)

**Reply by lauraneb** [reply]
Every time I want to launch an operation I have to manually bring the head back to 0.0.
as soon as I try to start the bed tramming, it comes to the center of the plate and starts its bed tramming from the center to move to the right and try to get out of the frame of the machine, forcing the motor.
